package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "ibm-cloud-databases",
+  "version": "0.0.1",
+  "description": "IBM Cloud Databases Node.js SDK",
+  "repository": {
+    "type": "git",
+    "url": "https://github.ibm.com/ibmcloud/icd-node-sdk"
+  },
+  "keywords": [
+    "ibm"
+  ],
+  "author": "IBM Corp.",
+  "scripts": {
+    "eslint:fix": "eslint . --fix",
+    "eslint:check": "eslint . --cache",
+    "tslint:fix": "tslint --fix -p . -c tslint.json",
+    "tslint:check": "tslint -p . -c ./tslint.json",
+    "lint": "npm run eslint:check && npm run tslint:check",
+    "lint-fix": "npm run eslint:fix && npm run tslint:fix",
+    "build": "tsc && cp package.json dist/",
+    "prepublishOnly": "npm run build",
+    "postversion": "tsc-publish --no-checks --dry-run",
+    "jest": "jest",
+    "test": "npm run build && npm run lint && jest test/",
+    "test-unit": "npm run build && jest test/unit/",
+    "test-integration": "npm run build && jest test/integration",
+    "test-examples": "npm run build && jest examples/",
+    "test-travis": "jest --runInBand --testNamePattern='^((?!@slow).)*$' test/",
+    "test-unit-travis": "jest --runInBand test/unit/",
+    "test-integration-travis": "jest --runInBand --no-colors --testNamePattern='^((?!@slow).)*$' --json test/integration > test-output.log",
+    "report-coverage": "codecov",
+    "check-packages": "installed-check -e -d -v"
+  },
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=10.0.0"
+  },
+  "dependencies": {
+    "@types/node": "^12.0.8",
+    "extend": "^3.0.2",
+    "ibm-cloud-sdk-core": "^2.10.2"
```

Comment:
This doesn't affect much because the `^` is already there but we recently released `2.10.3`, may be good to go ahead and bump this version up